staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -77,8 +77,6 @@ type Event struct {
 	// AuditLevel at which event was generated
 	Level Level
 
-	// Time the request reached the apiserver.
-	Timestamp metav1.Time
```

Comment:
@CaoShuFeng Just in case, I'm always ready to discuss and change my opinion, feel free to argue, I'll be happy to challenge all ideas :)

And by the way, thanks a lot for working on this!